tldw_Server_API/app/core/AuthNZ/reset_admin_password.py

```
@@ -0,0 +1,104 @@
+
+        # Hash new password
+        password_service = PasswordService()
+        new_hash = password_service.hash_password(new_password)
+
+        # Update password in database (works for both SQLite and PostgreSQL)
+        await users_db.update_user(user_id, password_hash=new_hash)
+
+        print(f"[reset-password] Password updated for user '{username}' (id={user_id}, role={user_role}).")
+        return True
+
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)